tests/python/shared/assets/cvat_db/data.json

```diff
@@ -6180,54 +6180,6 @@
     "frames": "[]"
   }
 },
-{
-  "model": "engine.segment",
-  "pk": 38,
-  "fields": {
-    "task": 29,
-    "start_frame": 0,
-    "stop_frame": 7,
-    "chunks_updated_date": "2024-10-02T08:13:16.623Z",
-    "type": "range",
-    "frames": "[]"
-  }
-},
-{
-  "model": "engine.segment",
-  "pk": 39,
-  "fields": {
-    "task": 29,
-    "start_frame": 8,
-    "stop_frame": 15,
-    "chunks_updated_date": "2024-10-02T08:13:16.623Z",
-    "type": "range",
-    "frames": "[]"
-  }
-},
-{
-  "model": "engine.segment",
-  "pk": 40,
-  "fields": {
-    "task": 29,
-    "start_frame": 16,
-    "stop_frame": 22,
-    "chunks_updated_date": "2024-10-02T08:13:16.623Z",
-    "type": "range",
-    "frames": "[]"
-  }
-},
-{
-  "model": "engine.segment",
-  "pk": 41,
-  "fields": {
-    "task": 29,
-    "start_frame": 23,
-    "stop_frame": 28,
-    "chunks_updated_date": "2024-10-02T08:13:16.623Z",
-    "type": "range",
-    "frames": "[]"
-  }
-},
```

Comment:
It is the result of `docker exec test_cvat_server_1 python manage.py dumpdata --indent 2 --natural-foreign --exclude=auth.permission --exclude=contenttypes --exclude=django_rq > tests/python/shared/assets/cvat_db/data.json`. I guess it happened because these segments were duplicated - there still are segments with pk 38-41 in data.json